app/views/recipes/show.html.erb

```diff
@@ -0,0 +1,4 @@
+<h1><%= "#{@recipe.name}" %></h1>
+  <h3>Name:<%= @recipe.name %></h3>
+  <h3>Complexity:<%= @recipe.complexity %></h3>
```

Comment:
Nice job getting a recipes name, complexity, and genre. Looks like you didn't quite get to listing all ingredients in this recipe. If we think back to reaching through objects, ingredients will exist on the `recipe` object because of the many-to-many join. So you would be able to iterate over a recipes ingredients to get the name.
```ruby
<% @recipe.ingredients.each do |ingredient| %>
   <li><%= ingredient.name %></li>
<% end %>
```